ra2ce/graph/networks.py

```diff
@@ -80,12 +81,25 @@ def __init__(self, config, files):
             self.region_var = None
 
         # Cleanup
-        self.snapping = config["cleanup"]["snapping_threshold"]
-        self.segmentation_length = config["cleanup"]["segmentation_length"]
+        _cleanup_options = config.get("cleanup", {})
+        if any(_cleanup_options.items()) and any(
+            [v for k, v in _cleanup_options.items()]
+        ):
+            _cleanup_options["cleanup"] = True
+        self._setup_cleanup_options(_cleanup_options)
 
         # files
         self.files = files
 
+    def _setup_cleanup_options(self, opt: dict = {}):
```

Comment:
Don't use default values (i'm forbidding everyone from doing this :)). Let the user explicitly set the value for a method argument.
```suggestion
    def _setup_cleanup_options(self, opt: dict):
```

Reply:
Got it